1.10/installing/custom/system-requirements/index.md

```diff
@@ -102,13 +102,14 @@ High speed internet access is recommended for DC/OS installation. A minimum 10 M
 
 Docker must be installed on all bootstrap and cluster nodes. The supported versions of Docker are:
 
+- 17.05.0
 - 1.13.x
 - 1.12.x
 - 1.11.x
 
 **Recommendations**
 
-* Docker 1.11.x - 1.13.x is recommended <a href="https://github.com/docker/docker/issues/9718" target="_blank">for stability reasons</a>.
+* Docker 1.11.x and above is recommended <a href="https://github.com/docker/docker/issues/9718" target="_blank">for stability reasons</a>.
```

Comment:
I would say 1.11.x - 17.05.0 since we aren't automatically testing newer versions.

Reply:
Updated.